mix.exs

```diff
@@ -14,13 +16,21 @@ defmodule Wallaby.Mixfile do
      description: "Concurrent feature tests for elixir",
      deps: deps(),
      docs: docs(),
-     test_coverage: [tool: ExCoveralls],
+
+     # Custom testing
+     aliases: ["test.all": ["test", "test.drivers"],
+               "test.drivers": &test_drivers/1],
```

Comment:
I agree with having `test` run everything and then `test.unit` and `test.integration` run unit and integration tests, respectively. However, I'm worried about overwriting the built-in `test` task. If you have any examples of how to safely improve the names of these tasks, I'd be super happy to make the changes.